frontend/apps/filemanager/filemanagercollate.lua

```diff
@@ -0,0 +1,171 @@
+local datetime = require("datetime")
+local DocSettings = require("docsettings")
+local ffiUtil = require("ffi/util")
+local sort = require("sort")
+local util = require("util")
+local _ = require("gettext")
+
+local FileManagerCollate = {
+    collate_by_id = nil
+}
+
+function FileManagerCollate.getCollates()
```

Comment:
Wouldn't it be easier to just make that a real table instead of this weird thin wrapper around a table?

e.g.,

FileManagerCollate.collate_list = {
    {
        id = "blah",
        ...
    },
}
```

Use patches can then just `table.append(FileManagerCollate.collate_list)` instead of wrapping a function?

Reply:
I agree, the wrapper is a remnant my experiments.